swagger.yaml

```diff
@@ -222,6 +222,19 @@ definitions:
       data:
         description: Configuration for data transfer
         $ref: '#/definitions/data-configuration'
+      imageRepositories:
+        description: Image repositories configuration
+        $ref: '#/definitions/image-repositories'
+
+  image-repositories:
+    type: object
+    properties:
+      authFile:
+        type: string
+        description: |
```

Comment:
this doesn't rendered well in https://github.com/jakub-dzon/k4e-operator/blob/d83e758314c2e26418e69ef4f5add18c9059ecd1/docs/http_api_swagger.md#-image-repositories
perhaps there is a need to avoid new lines in text.

Reply:
Good catch! I haven't paid attention to the rendering of that MD file.

Reply:
Done.